packages/utilities/src/utils/schema-path.js

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache2.0
  */
 
+// necessary to get exceptions thrown for attempts to modify frozen objects
```

Comment:
Picky, but for comment formatting consistency:

```suggestion
// Necessary to get exceptions thrown for attempts to modify frozen objects.
```